_source/_posts/2022-09-09-supabase-vue.md

```diff
@@ -0,0 +1,937 @@
+---
+layout: blog_post
+title: ""
+author:
+by: advocate|contractor
+communities: [devops,security,mobile,.net,java,javascript,go,php,python,ruby]
+description: ""
+tags: []
+tweets:
+- ""
+- ""
+- ""
+image:
+type: awareness|conversion
+---
+
+## Introduction
+
+Supabase is an open-source Firebase alternative. It provides several critical functions for modern web applications such as Postgres databases, authentication, real-time subscriptions, and blob storage. The interface is very easy to use but is very feature-rich. Supabase also provides support for javascript via a client that is available via yarn or npm. Like the web platform, the client is very easy to use and integrate into any modern web application. Best of all it is free to get started with supabase and you can scale into a premium plan on demand.
+
+In this article, you will learn how to build a wellness tracker app that will allow authenticated users to create new entries for specified days. These entries will be marked as public or private. Public entries will be visible to all visitors however private entries are only available to authenticated users.
+
+The application will be built with Vue.js using tailwinds to style the site. You will use Supabase to store the data used for the site. Finally, you will use Okta to secure the application.
+
+## Prerequisites
```

Comment:
The first couple of headings here need some work. They're not capitalizing on the keywords for SEO.

Reply:
Also, any idea why the "Deploy Preview" is failing?

Reply:
@havi there is a build error. It looks like it's missing the `raw` liquid tag to me, but I didn't try running locally.

@nickolasfisher please build locally and fix the error. You can build via [Docker or local install](
https://github.com/oktadev/okta-blog#requirements).

Reply:
@alisaduncan  https://github.com/oktadev/okta-blog/pull/1264/commits/ecd62a07caf955dfa2e68bdfd9feba1dd3a81e09 should resolve that